internal/runner/runner.go

```diff
@@ -73,7 +76,8 @@ func FreshRunner (current *Runner) *Runner {
 		Target: current.Target,
 		NodeID: current.NodeID,
 		Cache: &Cache{},
-		CDS: &CDS{},
+		CDS: &Service{},
+		LDS: &Service{},
```

Comment:
I like how this leaves an obvious and  hopefully simple path for yet defined services.